packages/core/src/transport/httpRequest.ts

```diff
@@ -67,6 +77,14 @@ function addBatchTime(url: string) {
   return `${url}${url.indexOf('?') === -1 ? '?' : '&'}batch_time=${new Date().getTime()}`
 }
 
+function addRequestId(url: string) {
+  return `${url}${url.indexOf('?') === -1 ? '?' : '&'}dd-request-id=${generateUUID()}`
+}
+
+function isEndpointIntakeV2(url: string) {
+  return includes(url, '/api/v2')
```

Comment:
NIT: Since you are using `includes` here, maybe you could use it instead of `indexOf` above? (in addBatchTime too)